packages/bundle-size/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@fluentui/bundle-size",
+  "version": "1.0.0",
+  "private": true,
+  "bin": {
+    "bundle-size": "./bin/bundle-size.js"
+  },
+  "scripts": {
+    "lint": "just-scripts lint",
```

Comment:
Also a late comment (sorry) but since you're not using just for anything else, you might as well remove the just config and call eslint directly.

Reply:
Hm... Nice, also applied in #18322 👍